src/main/java/io/vlingo/actors/plugin/mailbox/telemetry/DefaultMailboxTelemetry.java

```diff
@@ -38,7 +38,8 @@ public void onPullEmptyMailbox() {
 
   @Override
   public void onPulledMessage(final Message message) {
-
+    actorGaugeFor(message).decrementAndGet();
+    actorClassGaugeFor(message).decrementAndGet();
```

Comment:
Do you have a brief glossary of what each gauge is?

Reply:
The [concepts](https://micrometer.io/docs/concepts) section of the micrometer documentation is amazing, if you want to take a look.

If you mean more specifically those that are coded there. For each actor, there are two gauges (number of items in the current timestamp):

* Gauge by actor: All pending messages by each actor instance of a class. For example, there will be a gauge of the number of pending messages in the mailbox for each Actor of class Pinger.
* Gauge by class: All pending messages by actor class. For example, a sum of all mailbox pending messages for the class Pinger.